src/sass/static/pages.scss

```diff
@@ -1,4 +1,80 @@
 // styling for static pages that are too short or simple to need their own styling page
+.ib-signup-content {
+    margin-bottom: 0 !important;
+}
+
+.faq-page {
+    h3 {
+        color: $COLOR_ORANGE;
+
+        &:not(:first-of-type) {
+            margin-top: 3rem;
+        }
+    }
+    @media screen and (max-width: 480px) {
+        .sidebar-container {
+            margin-bottom: 2rem;
+
+            .sidebar a {
+                line-height: 2.5rem;
+            }
+        }
+    }
+}
+
+.signup-page {
```

Comment:
since it's about affiliate signup page, I would recommend a clearer class name, since we also have binary account signup page and mt account sign up page. also the block is pretty long so it would make sense to keep it in its own file as before.

Reply:
these styles are used by both affiliate and IB sign up pages, i guess i can use specific classes for them and declare styles like this: `.affiliate-signup, .ib-signup {`
But then do i keep it in pages.scss? 🤔

Reply:
I think it's still kind of an affiliate page, so should be fine in affiliate file :P but yeah, using different class names for each and assigning the same style to both sounds good.